cmd/build.go

```diff
@@ -42,44 +44,69 @@ func Build(ctx context.Context) *cobra.Command {
 			}
 
 			path := "."
-			if len(args) == 1 {
-				path = args[0]
-			}
 
-			if err := utils.CheckIfDirectory(path); err != nil {
-				return fmt.Errorf("invalid build context: %s", err.Error())
+			if options.Tag != "" {
+				if len(args) == 1 {
+					path = args[0]
+				}
+				err := buildImageWithOpts(options, path)
+				if err != nil {
+					return err
+				}
 			}
-			options.Path = path
 
-			if options.File == "" {
-				options.File = filepath.Join(path, "Dockerfile")
-			}
+			manifestPath := contextCMD.GetOktetoManifestPath(path)
+			if manifestPath != "" {
+				buildManifest, err := model.GetBuildManifest(manifestPath)
+				if err != nil {
+					return err
+				}
+				if len(buildManifest) != 0 {
+					image := ""
```

Comment:
I would call this service. `image` is more the image name we will build and push to the registry